tests/unit/issuing.test.js

```diff
@@ -33,471 +44,471 @@ function getSampleCredential(signed = false) {
-    expect(results.presentationResult.results[0].proof.type).toBe('EcdsaSecp256k1Signature2019');
-    expect(results.presentationResult.results[0].proof.challenge).toBe('some_challenge');
-    expect(results.presentationResult.results[0].proof.domain).toBe('some_domain');
-    expect(results.presentationResult.results[0].proof.proofPurpose).toBe('authentication');
-    expect(results.presentationResult.results[0].proof.verificationMethod).toBe(keyUrl);
-    expect(results.presentationResult.results[0].verified).toBe(true);
-    expect(results.credentialResults[0].verified).toBe(true);
-    expect(results.credentialResults[0].results).toBeDefined();
-    expect(results.credentialResults[1].verified).toBe(true);
-    expect(results.credentialResults[1].results).toBeDefined();
-  }, 30000);
-});
-
-describe('Verifiable Credential incremental creation', () => {
-  test('VC creation with only id should be possible, yet bring default values', async () => {
-    const credential = new VerifiableCredential(sampleId);
-    expect(credential.id).toBe(sampleId);
-    expect(credential.context).toEqual(['https://www.w3.org/2018/credentials/v1']);
-    expect(credential.type).toEqual(['VerifiableCredential']);
-    expect(credential.issuanceDate).toEqual(expect.anything());
+}];
+
+testingKeys.forEach((testKey) => {
+  const { sigType, keyDocument } = testKey;
+  const keyUrl = keyDocument.id;
+  const controllerUrl = keyDocument.controller;
+
+  describe(`Verifiable Credential Issuing [${keyDocument.type}]`, () => {
+    test('Issuing should return an object with a proof, and it must pass validation.', async () => {
+      const credential = await issueCredential(testKey.keyDocument, getSampleCredential());
+      expect(credential.id).toBe('https://example.com/credentials/1872');
+      expect(credential.type).toContain('VerifiableCredential');
+      expect(credential.type).toContain('AlumniCredential');
+      expect(credential.issuanceDate).toBe(issuanceDate);
+      expect(credential.credentialSubject.id).toBe('did:example:ebfeb1f712ebc6f1c276e12ec21');
+      expect(credential.credentialSubject.alumniOf).toBe('Example University');
+      expect(credential.issuer).toBe(controllerUrl);
+      expect(credential.proof.type).toBe(sigType);
+      expect(credential.proof.created).toBeDefined();
+      expect(credential.proof.proofPurpose).toBe('assertionMethod');
+      expect(credential.proof.verificationMethod).toBe(keyUrl);
+
+      const result = await verifyCredential(credential);
+      expect(result.verified).toBe(true);
+      expect(result.results[0].proof).toBeDefined();
+      expect(result.results[0].proof.proofPurpose).toBe('assertionMethod');
+      expect(result.results[0].proof.type).toBe(sigType);
+      expect(result.results[0].proof.verificationMethod).toBe(keyUrl);
+      expect(result.results[0].verified).toBe(true);
+    }, 30000);
+
+    // JsonWebKey signatures cant use proofValue
```

Comment:
why not? sorry if it's a dumb question - I haven't looked into JWK's at all

Reply:
because the jsonwebkey signed only produces jws signatures, propfValue isn't one and can't be represented as one